src/components/controls/km-input-lstring.vue

```diff
@@ -38,8 +22,9 @@ import { without } from "lodash";
 
 const model = defineModel({type:Object, required: true, default:{}})
 const props = defineProps({
-  locales: { type: Array, required: true },
+  locales:  { type: Array, required: true },
   disabled: { type: Boolean, default: false },
+  rows:     { type: Number, required: false, default: 1 }
```

Comment:
can you add placeholder property as well

Reply:
done